docker/wiki.fly.nginx.conf

```diff
@@ -0,0 +1,14 @@
+server {
+    listen 80 default_server;
+    root /var/www/public;
+
+    index index.php index.html index.htm;
+
+    try_files $uri /index.php?$args;
+
+    location ~ \.php$ {
+        fastcgi_pass 127.0.0.1:9000;
```

Comment:
i copied the file you created and changed this one line.  In the fly.io dockerfile below I combine the php server and nginx server into one docker image.  This simplifies the deployment architecture and keeps me in the free tier.

We should probably use this version of the code for docker compose too

Reply:
oh cool I didn't know you could do that, I thought I read somewhere that there could be only one server per docker container. yeah let's change this for the main one too.

Reply:
it's more idiomatic the way you did it, but then harder to manage since not all tools play nice with docker-compose